package.json

```diff
@@ -31,7 +34,18 @@
     "prettier": "^2.8.7",
     "typescript": "^5.0.4"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@onflow/fcl": "^1.3.2",
```

Comment:
I guess `fcl` should rather be a `peerDependency`, allow this lib to share dapp's fcl lib. For the sake of development fcl would remain a devDependency. The way of declaring peer deps can be seen e.g. on this random package I found: https://github.com/twbs/bootstrap/blob/main/package.json#LL103C7-L103C15